docs/src/layouts/AppFooter.tsx

```diff
@@ -175,10 +137,51 @@ export default function AppFooter() {
         </Box>
       </Box>
       <Divider />
-      <Box sx={{ py: 4 }}>
+      <Box
+        sx={{
+          py: 4,
+          display: { xs: 'block', sm: 'flex' },
+          alignItems: { sm: 'center' },
+          justifyContent: { sm: 'space-between' },
+        }}
+      >
         <Typography color="text.secondary" variant="body2">
-          Currently v4.11.0. Released under the MIT License. Copyright © 2020 Material-UI SAS.
+          Copyright © {new Date().getFullYear()} Material-UI SAS.
```

Comment:
Not sure what we should use here, the name of the project

```suggestion
          Copyright © {new Date().getFullYear()} MUI.
```

or the legal entity, that is, I believe in uppercase
```suggestion
          Copyright © {new Date().getFullYear()} MATERIAL-UI SAS.
```

cc @mbrookes